antenna/configlib.py

```diff
@@ -180,38 +193,49 @@ def _initialize_environments(self):
         # Add OS environment
         envs.append(ConfigOSEnv())
 
-        # Add .ini environment
-        fn = os.environ.get('ANTENNA_INI', 'antenna.ini')
-        if os.sep not in fn:
-            fn = os.path.join(os.getcwd(), fn)
+        # Add .ini environment if they specify ANTENNA_INI in the environment
+        fn = os.environ.get('ANTENNA_INI')
+        if fn:
+            if os.sep not in fn:
+                fn = os.path.join(os.getcwd(), fn)
```

Comment:
Regarding your first comment, you're right on. However, we change `fn` here so when we raise the configuration error later, it has the complete filename that was used. If we don't do that, then the IOError isn't helpful--the programmer is left wondering why the program can't find the ini file when it's *right there*!

Regarding your second comment, that's interesting.... How does it help to have both those cases be IOErrors rather than one a ConfigurationError and the other an IOError? Is there a situation where we'd want to catch these exceptions and do something more helpful?